python/ray/autoscaler/aws/defaults.yaml

```diff
@@ -1,13 +1,9 @@
 # An unique identifier for the head node and workers of this cluster.
 cluster_name: default
 
-# The minimum number of workers nodes to launch in addition to the head
-# node. This number should be >= 0.
-min_workers: 0
```

Comment:
(question) What's the reason for removing this?

Reply:
This is not used anywhere in the code. We have multi node types. Every node_type has its own min_workers. the global min_workers is useless.